lib/tasks/fake_data.rake

```diff
@@ -0,0 +1,13 @@
+require "fake_data"
+
+namespace :fake_data do
+  desc "Insert fake data for testing the system"
+  task :insert, [] => :environment do
```

Comment:
Do we want any ability to pass in number we want to create? not sure if we want to be able to start small and then have to scope to experiment with an even bigger number?

Reply:
Agreed, I've added this in 226c296 allowing the user to specify a proportion of data.

Reply:
ooh a proportion is clever!